pepper-apis/src/main/java/org/broadinstitute/ddp/db/ActivityInstanceDao.java

```diff
@@ -340,6 +340,8 @@ public Long getActivityInstanceIdByGuid(Handle handle, String guid) {
                             createdAtMillis, statusTypeCode, isActivityWriteOnce, isActivityInstanceReadonly);
                     long createdAt = rs.getLong(SqlConstants.ActivityInstanceTable.CREATED_AT);
                     boolean isFollowup = rs.getBoolean(SqlConstants.StudyActivityTable.IS_FOLLOWUP);
+                    boolean isHidden = rs.getBoolean(SqlConstants.ActivityInstanceTable.IS_HIDDEN);
+                    // todo: handle isHidden
```

Comment:
Is this todo relevant anymore?

Reply:
It's fine to leave this here -- I have removed it in the follow-up PRs.